src/Template/Sentences/show.ctp

```diff
@@ -149,7 +147,11 @@ echo $this->element('/sentences/navigation', [
             echo $this->Sentences->originText($sentence);
         }
 
-        if (!empty($contributions)) {
+        if ($contributions->count() > 0) {
+            if (!empty($sentence->translations)) {
```

Comment:
You're probably going to hate me but I realized while testing that `$sentence->translations` will (should?) never be empty. It's structured into two arrays.
```
[
	(int) 0 => [], // direct translations
	(int) 1 => [] // indirect translations
]
```
So we need to test that `$sentence->translations[0]` is empty.

Reply:
No problem, I can tolerate a lot of criticism (especially if it's justified) :-)

You probably won't believe me but I originally wrote the test with `$sentence->translations[0]` but just before committing I thought for some reason it's not necessary to be that specific and then forgot to also test for sentences without any translations where that makes a difference.